Nos/Views/GoldenPostView.swift

```diff
@@ -68,23 +68,36 @@ struct GoldenPostView: View {
             let parsedAttributedContent = await Event.attributedContent(
                 noteID: note.identifier,
                 context: backgroundContext
-            ) 
+            )
             withAnimation(.easeIn(duration: 0.1)) {
                 self.noteContent = .loaded(parsedAttributedContent)
             }
         }
     }
-    
+
     var isTextOnly: Bool {
-        (try? note.content?.findUnformattedLinks().count ?? 0) == 0
+        if let content = note.content {
+            return (try? URLParser().findUnformattedURLs(in: content).count) == 0
```

Comment:
Maybe this URLParser could be a property on GoldenPostView -- maybe even a `@Dependency`? (I'm still not sure how to make it available as an `@Dependency` but seems like it should be possible.)

Reply:
Nice, I added it as a dependency.